cookbooks/aws-parallelcluster-common/resources/efa/efa_redhat8.rb

```diff
@@ -20,6 +20,17 @@
 
 use 'partial/_setup'
 
+action :setup do
+  if node['cluster']['platform_version'].to_f < 8.4
```

Comment:
Question: why is `node['cluster']['platform_version']` used instead of `node['platform_version']`?

Reply:
`node['platform_version']` is kind of a reserved attribute, so it's not possible to override it (because it gets overridden by the actual test platform version) so I need to use a different parameter